holopy/core/tests/test_vis.py

```diff
@@ -244,6 +239,13 @@ def test_maintains_metadata(self):
 class ShowTest(unittest.TestCase):
     @attr("medium")
     def test_show(self):
+        try:
```

Comment:
My 2 cents:

I'd rather see the import at the top of the file, w/ a `unittest.skipIf(NO_MATPLOTLIB )` or something here. You can even do `from holopy.core.io.vis import _NO_MATPLOTLIB` I believe.

Reply:
agreed this is better. Had to check again though because it needs matplotlib.